packages/protocol/lib/web3-utils.ts

```diff
@@ -387,6 +393,28 @@ export function getFunctionSelectorsForContract(contract: any, contractName: str
   return selectors
 }
 
+export function checkInheritance(baseContractName: string, derivativeContractArtifact: any) {
```

Comment:
Can the artifact be typed better with types from the OpenZeppelin package?

Reply:
No because we also use truffle artifacts